manager-bundle/src/Resources/skeleton/public/index.php

```diff
@@ -23,6 +23,18 @@
     stream_wrapper_unregister('phar');
 }
 
+// System maintenance mode comes first as it has to work even if the vendor directory does not exist
+if (file_exists(__DIR__.'/maintenance.html')) {
```

Comment:
If the maintenance mode is enabled, anybody who calls `foobar.com/maintenance.html` will get the same as if they called `foobar.com/foobar`, yes. But who cares 🤷‍♂️

Reply:
No, they will not get the same, because `foobar.com/maintenance.html` will return status `200 OK` whereas `foobar.com/foobar` will return status `503 Service unavailable`. Can we not write the file to `var/maintenance` instead? Then every request will be answered with `503 Service unavailable`.